tests/src/Appender/RollingFileAppenderTest.cs

```diff
@@ -155,7 +157,17 @@ private static void VerifyFileCount(int iExpectedCount)
 		/// <param name="iExpectedCount"></param>
```

Comment:
Couldn't we count both files that match `c_fileName` and `c_fileNameWithFolder`? This way we would only have one `VerifyFileCount([...])` to invoke from the tests, just like there is one `DeleteTestFiles()`. This should work fine since the tests also remove both kinds of logfiles.